draft-ietf-taps-impl.md

```diff
@@ -1030,39 +1048,78 @@ Data Unit: Message
 API mappings for SCTP are as follows:
 
 Connection Object:
-: Connection objects represent a flow of SCTP messages between a client and a server, which may be an SCTP association or a stream in a SCTP association. How to map Connection objects to streams is described in {{NEAT-flow-mapping}}; in the following, a similar method is described.
-To map Connection objects to SCTP streams without head-of-line blocking on the sender
-side, both the sending and receiving SCTP implementation must support message interleaving {{!RFC8260}}.
-Both SCTP implementations must also support stream reconfiguration. Finally, both communicating endpoints
-must be aware of this intended multiplexing; {{NEAT-flow-mapping}} describes a
-way for a Transport System to negotiate the stream mapping capability using SCTP's adaptation layer indication,
-such that this functionality would only take effect if both ends sides are aware of it.
-The first flow, for which the SCTP association has been created, will always use stream id zero.
-All additional flows are assigned to unused stream ids in growing order. To avoid a conflict
-when both endpoints map new flows simultaneously, the peer which initiated the transport connection
-will use even stream numbers whereas the remote side will map its flows to odd stream numbers.
-Both sides maintain a status map of the assigned stream numbers. Generally, new streams
-must consume the lowest available (even or odd, depending on the side) stream number; this
-rule is relevant when lower numbers become available because Connection objects associated
-to the streams are closed.
+: Connection objects can be mapped to an SCTP association or a stream in an SCTP association.
+Mapping Connection objects to SCTP streams is called "stream mapping" and has additional requirements as follows.
+The following explanation assumes a client-server communication model.
+
+Stream mapping requires an association to already be in place between the client
+and the server, and it requires the server to understand that a new incoming stream
+should be represented as a new Connection Object by the Transport Services system.
+To signal the use of stream mapping, the server SHOULD use the adaptation layer indication
+parameter of LISTEN.SCTP and the client SHOULD use the adaptation layer indication
+parameter of CONNECT.SCTP, both with the "Stream Mapping" adaptation code point.
```

Comment:
To make my comment more concrete, I'm not convinced that the code point is necessary.

Reply:
Yeah, I think we have consensus on that. Consider it gone.